File: mario.py
from cs50 import get_int


def main():

    # get height function defined later
    height = get_height()

    # i for height of columbs
    for i in range(height):

        # s to add spaces befor first block creating right alligned pyramid
        for s in range(height - i - 1):
            print(" ", end="")

        # j for width of pyramid based on height (this is for the first pyramid)
        for j in range(i + 1):
            print("#", end="")

        print("  ", end="")
        # j for width of pyramid based on height (pyramid 2)
        for j in range(i + 1):
            print("#", end="")

        # for new line when done with entire row
        print()


def get_height():

    while True:

        # This Gets an int called height from the user
        height = get_int("Height: ")

        # This makes sure that the height int is between 1 and 8
        if (height > 0) and (height < 9):
            break

    return height


main()

File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n, s, i, j;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    //i for row
    for (i = 0; i < n; i++)
    {
        //s for space
        for (s = 0; s < n - i - 1; s++)
        {
            printf(" ");
        }
        //j for column
        for (j = 0; j <= i; j++)
        {

            printf("#");
        }
        //for space between pyramid
        printf("  ");
        //for second pyramid
        for (j = 0; j <= i; j++)
        {
            printf("#");
        }
    printf("\n");
    }
}